scripts/tmp.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I like the idea of tmp.sh and TMP.sh scripts, but let's find a better name (and subdirectory, probably). Maybe tmp/TPM is due to the name of the LM_TMP, but not very fortunate... as tpm.sh / TPM.sh is the typicall somebody will use for temporal scripts (as foo.sh or bar.sh.